README.md

```diff
@@ -243,6 +241,134 @@ async function run() {
     template: "TEMPLATE_NAME_OR_ID",
   });
   console.log(runId);
+
+  // Example: List notifications
+  const { paging, results } = await courier.notifications.list({});
+  console.log(results);
+
+  // Example: Get notification content
+  const { blocks, channels } = await courier.notifications.getContent(
+    "notification1"
+  );
+  console.log(blocks);
+  console.log(channels);
+
+  // Example: Get notification draft content
+  const { blocks, channels } = await courier.notifications.getDraftContent(
+    "notification1"
+  );
+  console.log(blocks);
+  console.log(channels);
+
+  // Example: Post notification variations
+  await courier.notifications.postVariations("notification1", {
```

Comment:
does this need a notification Id or it can also take a mapped event?

Reply:
It needs a notificationId